File: py/IEC61968/Assets/SealConditionKind.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106
# seal_condition_kind.py

from enum import Enum

class SealConditionKind(Enum):
    LOCKED = 1
    OPEN = 2
    BROKEN = 3
    MISSING = 4
    OTHER = 5

File: java/IEC61968/Assets/SealConditionKind.java
package IEC61968.Assets;


/**
 * Kind of seal condition.
 * @author T. Kostic
 * @version 1.0
 * @created 25-Dec-2023 8:45:24 PM
 */
public enum SealConditionKind {
	/**
	 * Seal is locked.
	 */
	locked,
	/**
	 * Seal is open.
	 */
	open,
	/**
	 * Seal is broken.
	 */
	broken,
	/**
	 * Seal is missing.
	 */
	missing,
	/**
	 * Other kind of seal condition.
	 */
	other
}